explorer/.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
Please revert this change because it's not related to the issue this PR is solving

Reply:
Sure, I will add it in a separate PR